src/app/dashboard/page.tsx

```diff
@@ -0,0 +1,221 @@
+'use client'
+
+import { createBrowserClient } from '@/utils/supabase'
+import { UUID } from 'crypto'
+import { useEffect, useState } from 'react'
+
+export default function Page() {
+  const [username, setUsername] = useState('')
+  const [events, setEvents] = useState<any[] | null>(null)
+  const [eventData, setEventData] = useState({
+    title: '',
+    description: '',
+    startTime: '',
+    endTime: '',
+    location: '',
+    timezone: '',
+    mode: '',
+    config: JSON,
+    creator: '',
+  })
+  const [submitMessage, setSubmitMessage] = useState('')
+  const supabase = createBrowserClient()
+
+  useEffect(() => {
+    const username = localStorage.getItem('username')
+    if (username) {
+      setUsername(username)
+    }
+    const getMyEvents = async (creatorId: UUID) => {
+      const { data, error } = await supabase
+        .from('Events')
+        .select()
+        .eq('creator', creatorId)
+
+      if (!error) {
+        setEvents(data)
+      } else {
+        console.error(error)
+      }
+    }
+
+    getMyEvents('9e33186f-95db-4385-a974-ee38c8e07547')
```

Comment:
Filler UUID until local storage to store a user ID on event creation or visiting an event page is setup so that we can use the user's id in local storage to get events isntead. I have added this specific UUID as an id for a user in the seed.sql so that it should be work with local databases.